test/pages/scripttag/index.html

```diff
@@ -7,73 +7,108 @@
+                <button id="trackLogin" style="width:100px;">Track Login</button>
+                <button id="trackLogout" style="width:100px;">Track Logout</button>
+            </div>
+        </div>
+    </div>
+    <div class="row">
+        <div class="column">
+            <h2>Encryption</h2>
+            <div class="block">
+                Name on Card: <input type="text" id="name" />
+                Card Number: <input type="text" id="number" />
+                Month: <select id="month">
+                    <option selected value="1">January</option>
+                    <option value="2">February</option>
+                    <option value="3">March</option>
+                    <option value="4">April</option>
+                    <option value="5">May</option>
+                    <option value="6">June</option>
+                    <option value="7">July</option>
+                    <option value="8">August</option>
+                    <option value="9">September</option>
+                    <option value="10">October</option>
+                    <option value="11">November</option>
+                    <option value="12">December</option>
+                </select><br/>
+                Year: <input type="number" id="year" />
+
+                <br/> <button id="encrypt" > Encrypt </button>
+            </div>
+            <h2>Output</h2>
+            <pre id="encryptionOutput"></pre>
+            <h2>Output Error</h2>
+            <pre id="encryptionOutputError"></pre>
+        </div>
+        <div class="column">
+            <h2>Fingerprint Info</h2>
+            <pre id="fingerprintOutput"></pre>
+            <h2>Fingerprint Error</h2>
+            <pre id="fingerprintError"></pre>
+        </div>
+        <div class="column">
+            <h2>Tracking Error</h2>
+            <pre id="trackingError"></pre>
+        </div>
+    </div>
 
     <script src="../common.js"></script>
     <script src="./ravelin.js"></script>
-    <script>
-        document.getElementById('key-form').onsubmit = function() {
-            setRSAKey();
-            return false;
-        };
-
-        function setRSAKey() {
-            output(function() {
-                ravelinjs.setRSAKey(document.getElementById('rsa-key').value);
-                return 'Key set // ' + (new Date);
-            });
-        }
-        setRSAKey();
-
-        document.getElementById('form').onsubmit = function() {
-            var month = document.getElementById('month');
-            output(function() {
-                return ravelinjs.encrypt({
-                    nameOnCard: document.getElementById('name').value,
-                    pan: document.getElementById('number').value,
-                    month: month.options[month.selectedIndex].value,
-                    year: document.getElementById('year').value,
-                });
-            });
-            return false;
-        };
-    </script>
+    <script>new TestPageSetup(ravelinjs);</script>
```

Comment:
This isn't an object.

```suggestion
    <script>TestPageSetup(ravelinjs);</script>
```